CompilerSource/compiler/compile.cpp

```diff
@@ -98,6 +98,9 @@ dllexport int compileEGMf(EnigmaStruct *es, const char* exe_filename, int mode)
   return current_language->compile(es, exe_filename, mode);
 }
 
+static bool run_game = true;
+dllexport void ide_handles_game_launch() { run_game = false; }
```

Comment:
@JoshDreamland I had to use ``dllexport`` because ``extern "C"`` does not eliminate mangling. I see you have this macro'd, is my solution correct?

Reply:
Yes.